src/hooks/core/chat/useFetchChannelHook.ts

```diff
@@ -206,7 +208,8 @@ const useFetchChannelHook = () => {
 
       const isDeletedMessageOrSelfChat = isDeletedMessage || isSelfChatChannel;
       const isDeletedOrHasDeletedMessage = isDeletedMessage || hasDeletedMessage;
-      const isCommunityHasDeletedMessage = isDeletedOrHasDeletedMessage && isCommunityChannel;
+      const isCommunityHasDeletedMessage =
+        isDeletedOrHasDeletedMessage && (isCommunityChannel || isCommunityInvitationChannel);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for determining `isCommunityHasDeletedMessage` has been updated to include `isCommunityInvitationChannel`. Ensure that this change aligns with your intended behavior for handling deleted messages in community invitation channels.


<!-- This is an auto-generated comment by OSS CodeRabbit -->